src/Components/Authentication/__tests__/__snapshots__/Authentication.test.js.snap

```diff
@@ -3,54 +3,70 @@
 exports[`Authentication when user has no permissions renders the NotAuthorized 1`] = `
 <div>
   <div
-    class="pf-c-empty-state ins-c-not-authorized "
+    class="pf-v5-c-empty-state"
   >
     <div
-      class="pf-c-empty-state__content"
+      class="pf-v5-c-empty-state__content"
     >
-      <svg
-        aria-hidden="true"
-        class="pf-c-empty-state__icon"
-        fill="currentColor"
-        height="1em"
-        role="img"
-        style="vertical-align: -0.125em;"
-        viewBox="0 0 448 512"
-        width="1em"
-      >
-        <path
-          d="M400 224h-24v-72C376 68.2 307.8 0 224 0S72 68.2 72 152v72H48c-26.5 0-48 21.5-48 48v192c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V272c0-26.5-21.5-48-48-48zm-104 0H152v-72c0-39.7 32.3-72 72-72s72 32.3 72 72v72z"
-        />
-      </svg>
-      <h5
-        class="pf-c-title pf-m-lg"
-        data-ouia-component-id="OUIA-Generated-Title-1"
-        data-ouia-component-type="PF4/Title"
-        data-ouia-safe="true"
+      <div
+        class="pf-v5-c-empty-state__header"
       >
-        You do not have access to Remote Host Configuration
-      </h5>
+        <div
+          class="pf-v5-c-empty-state__icon"
+        >
+          <svg
+            aria-hidden="true"
+            class="pf-v5-svg"
+            fill="currentColor"
+            height="1em"
+            role="img"
+            viewBox="0 0 448 512"
+            width="1em"
+          >
+            <path
+              d="M400 224h-24v-72C376 68.2 307.8 0 224 0S72 68.2 72 152v72H48c-26.5 0-48 21.5-48 48v192c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V272c0-26.5-21.5-48-48-48zm-104 0H152v-72c0-39.7 32.3-72 72-72s72 32.3 72 72v72z"
+            />
+          </svg>
+        </div>
+        <div
+          class="pf-v5-c-empty-state__title"
+        >
+          <h5
+            class="pf-v5-c-empty-state__title-text"
+          >
+            You do not have access to Remote Host Configuration
+          </h5>
+        </div>
+      </div>
       <div
-        class="pf-c-empty-state__body"
+        class="pf-v5-c-empty-state__body"
       >
         Contact your organization administrator(s) for more information or visit 
         <a
-          href="./settings/my-user-access"
+          href="./iam/my-user-access"
```

Comment:
Is this change intended?

Reply:
hm. this is a snapshot. I'm wondering where this comes from and if this is also in the actually rendered UI. And if it is actually correct.

Reply:
let me have a look

Reply:
This change came from this commit:
https://github.com/RedHatInsights/sed-frontend/pull/272/commits/1d4fe7cdbae4c010fa53ffc1f02cf6a58e6ded7a 
But I did not make any changes to `Authentication.js` here so I guess it came from a different commit and I just updated the snapshot here

I checked previous commits, and I only made relevant changes in those commits:
1. This is the only commit where changes were made to `Authentication.js` & `Authentication.test.js` files, but those changes came after the snapshot update you mentioned, so they couldn't have came up from this commit: https://github.com/RedHatInsights/sed-frontend/pull/272/commits/53a5bf9dbfe01af4f72327af0ccd5ddd6c455d4f#diff-a194d08568455b4d77eb5df13c98260410d3ed5375a39aee43331d657dd85489 
2. Here was the first snapshot update to the snapshot file you mentioned, but I did not make any previous changes to `Authentication.js` & `Authentication.test.js` files, so it must have been caused from a previous package update, but I am not sure which one caused this change.
https://github.com/RedHatInsights/sed-frontend/pull/272/commits/f01bac6dd852a4b70b68d4b23200ea39f1349426#diff-f6381abb627b767d59d1844907725850526ea230abf57154c7492a365c0e5a20

Maybe this change came from the `config/setupTests.js` change in the same commit ? or a previous package update ?

Reply:
I found it in the UI

This is the page it is found on:
![image](https://github.com/RedHatInsights/sed-frontend/assets/93318917/76951840-6634-4e18-959d-0e18829c963d)

And this is the link it sends you to when you click [My User Access](https://stage.foo.redhat.com:1337/iam/my-user-access):
![image](https://github.com/RedHatInsights/sed-frontend/assets/93318917/08555441-2025-4e3e-9337-75fdf929c996)

It seems to me that this is the functionality we want when we click this link @bastilian

Reply:
That first screenshot is what we need to show in Activation Keys when a user doesn't have access.

If you need help with any of the Activation Key pieces let us know and we can take a look.

Reply:
I believe that is the correct link  - I don't even think that's a component that we control.

settings/my-user-access breaks anyway as far as I can see, so the iam one seems right